subsys/net/l2/ethernet/gptp/Kconfig

```diff
@@ -226,4 +226,37 @@ config NET_GPTP_STATISTICS
 	  Enable this if you need to collect gPTP statistics. The statistics
 	  can be seen in net-shell if needed.
 
+config NET_GPTP_SERVO_STEP_THRESHOLD
+	int "Set slave clock jump threshold (ns)"
+	default 1000000000
+	help
+	  In slave port mode, if the error with the master clock is greater
+	  than the value, time jump is required to prevent the clock from
+	  converging too slowly.
+
+config NET_GPTP_SERVO_MAX_ADJFREQ
+	int "Set servo maximum frequency adjust (Hz)"
+	default 25000000
+	help
+	  The maximum offset the servo will correct by changing the clock
+	  frequency, 25MHz is the minimum requirement of IEEE 802.1AS, If
+	  your device supports higher frequencies, set higher values to
+	  speed up convergence.
+
+config NET_GPTP_SERVO_KP
```

Comment:
As I read the code properly, this is now operating in phase domain. If so, could you describe it that way?